docker/lwua-dereferencer/dereferencer-py/dereferencer/dereference.py

```diff
@@ -1,128 +1,70 @@
 
 
-def isExpired(lastmod: datetime, cache_lifetime: int):
-    # check if the cache_lifetime is set
-    if cache_lifetime == 0:
-        return True
-    # check if the lastmod is set
-    if lastmod is None:
-        return True
-    # check if the lastmod is older than the cache_lifetime
-    if lastmod + timedelta(minutes=cache_lifetime) < datetime.now():
-        return True
-    # if the lastmod is not older than the cache_lifetime, return false
-    return False
-
-
 class Dereference:
     def __init__(self):
         pass
 
+    def init_targetstore(self, url):
+        try:
+            return TargetStore.TargetStore(url)
+        except Exception as e:
+            log.error("init targetstore failed, trying again in 1s")
+            log.error("error: {e}")
+            time.sleep(1)
+            self.init_targetstore(url)
+
     def run_dereference(self):
         log.info("running dereference")
 
         config_folder_path = config_path_from_config()
-        log.info(f"run_dereference on config files in {config_folder_path}")
-        # get all the config files in the config folder
-        # the files should be in yml or yaml format and should start with
-        # dereference
-        config_files = [f for f in config_folder_path.glob("dereference*.yml")]
+        log.info(f"run_dereference on config files in {str(config_folder_path)}")
+
+        config_files = [f for f in config_folder_path.glob("*.yml")]
         log.info(f"config files found: {config_files}")
 
-        # get all tasks from the graphdb
-        tasks_run = get_registry_of_lastmod()
+        for f in config_files:
+            with open(f, "r") as file:
+                content = file.read()
 
-        # for each config file , parse the file and get the config
-        for config_file in config_files:
-            log.info(f"running dereference for {config_file}")
-            derefTask = DerefTask(config_file)
+            # Log the content of the file
+            logging.info(content)
 
-            # log the tasks that have been run
-            log.info(f"tasks_run: {tasks_run}")
+        TARGETSTORE = TargetStore.TargetStore(GDB_URL)
 
-            if derefTask.file_name not in tasks_run:
-                log.info(f"task {derefTask.file_name} not in tasks_run")
-                derefTask.run_deref_task()
-                continue
+        log.info(TARGETSTORE)
 
-            # check if the task is expired
-            if isExpired(tasks_run[derefTask.file_name], derefTask.cache_lifetime):
-                derefTask.run_deref_task()
-                continue
+        CONFIGBUILDER = TravHarvConfigBuilder.TravHarvConfigBuilder(
+            TARGETSTORE,
+            str(config_folder_path),
+        )
 
-            log.info(
-                f"task {derefTask.file_name} was present in GraphDB and is not expired"
-            )
+        CONFIGLIST = CONFIGBUILDER.build_from_folder()
 
+        for travHarvConfig in CONFIGLIST:
```

Comment:
pythonic capitalization is recommended: trav_harv_config, trav_harv_config.prefix_set, trav_harv_config.config_name